app/views/testimonials/_form.html.erb

```diff
@@ -0,0 +1,76 @@
+
+
+
+<!-- 16:9 aspect ratio -->
+<div class="text-center">
+<div class="videoWrapperOuter">
+  <div class="videoWrapperInner">
+    <iframe src="https://www.youtube.com/embed/CR62t5lPrJ0"
+      frameborder="0" allowfullscreen></iframe>
+  </div>
+</div>
+
+ <img class="img-circle" width="200" height="200" src="https://www.codaisseur.com/assets/codaisseur-logo-9a3b4e4a4929449d09c6504c4d67d2dfc4abbed8d8f4333c8cf441dc63d49a11.svg">
+ <h3>What people say about TOC</h3>
+ <p>testimonials</p>
+</div>
+
+<section id="carousel">
+	<div class="container">
```

Comment:
please fix your indenting. Use 2 spaces consistently in all files

In this file you use 0 spaces(line 6), 2 spaces, 1 space(line 13), tabs (line 19) and lots of spaces(line 25).

Your colleagues will frown upon you...